recipes/nodejs_from_source.rb

```diff
@@ -24,9 +24,15 @@
 
 case node['platform_family']
 when 'rhel', 'fedora', 'amazon'
-  package %w(openssl-devel python3 tar)
+  directory "node['nodejs']['tmpdir']"
+  package %w(openssl-devel python2 tar)
+  link "#{node['nodejs']['tmpdir']}/python" do
+    to '/usr/bin/python2'
+  end
+  path = ENV['PATH']
+  ENV['PATH'] = "#{path}:#{node['nodejs']['tmpdir']}"
```

Comment:
Instead of doing this, I recommend you add the following line below in the `ark` resource:
```
environment(PYTHON: 'python2')
```